Scripts/whitesource-spdx.yml

```diff
@@ -0,0 +1,56 @@
+name: WhiteSource scan and SPDX report 
+
+on: 
+  push:
+    branches: []
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Build with Maven
+        run: mvn clean install -DskipTests=true
+
+      - name: WhiteSource Unified Agent Scan
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          WS_APIKEY: ${{secrets.APIKEY}}
+          WS_USERKEY: ${{secrets.USERKEY}}
+          WS_WSS_URL: https://saas.whitesourcesoftware.com/agent
+          WS_PRODUCTNAME: GH_${{github.event.repository.name}}
+          WS_PROJECTNAME: ${{github.ref}}_SPDX_report
+          WS_GENERATEPROJECTDETAILSJSON: true
+        run: |
+          curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
+          echo Unified Agent downloaded successfully
+          java -jar wss-unified-agent.jar -d ./
+
+      - name: Generate WhiteSource SPDX Report
+        env:
+          WS_WSS_URL: https://saas.whitesourcesoftware.com/agent
+        run: |
+            export WS_PROJECTTOKEN=$(jq -r '.projects | .[] | .projectToken' ./whitesource/scanProjectDetails.json)
+            echo WS_PROJECTTOKEN: $WS_PROJECTTOKEN
+            export WS_URL=$(echo $WS_WSS_URL | awk -F "agent" '{print $1}')
+            echo WS_URL: $WS_URL
+            echo === Installing WhiteSource SBOM Generator ===
+            pip  install ws_sbom_generator
+            echo === Generating SBOM ===
+            ws_sbom_generator -u ${{secrets.USERKEY}} -k ${{secrets.APIKEY}} -s $WS_PROJECTTOKEN -a $WS_URL -t tv -o ./whitesource-spdx
+
+      - name: Publish WhiteSource SPDX Report
+        uses: actions/upload-artifact@master
+        with:
+           name: WhiteSource SPDX report
```

Comment:
why not just copy into the whitesource logs folder and upload.  That way you have both?  
```
      - name: 'Upload WhiteSource folder'
        uses: actions/upload-artifact@v2
        with:
          name: WhiteSource
          path: whitesource
          retention-days: 1

```

Reply:
The intent is to save only the SPDX file.  

This will miss the whitesource-spdx folder and only get the logs.